### ISSUE TITLE
Just not working

### ISSUE THREAD

**Issue by oelbaganwg** [ryanto/next-s3-upload#48]
This seemed so promising, but a simple copy and paste doesn't work. using the basic file example. When you click upload a file and upload an image there's just no response, nothing in the console, no state change, nothing happens the page stays as is.

Simply using the demo code:

```
import { useState } from 'react';
import { useS3Upload } from 'next-s3-upload';

export default function UploadTest() {
  let [imageUrl, setImageUrl] = useState();
  let { FileInput, openFileDialog, uploadToS3 } = useS3Upload();

  let handleFileChange = async file => {
    let { url } = await uploadToS3(file);
    setImageUrl(url);
  };

  return (
    <div>
      <FileInput onChange={handleFileChange} />

      <button onClick={openFileDialog}>Upload file</button>

      {imageUrl && <img src={imageUrl} />}
    </div>
  );
}
```

**Comment by montanaflynn** [comment]
Same issue but it's intermittent and I can't always reproduce it. The issue appears to be that `handleFileChange` just doesn't fire after selecting a file sometimes.

**Comment by ryanto** [comment]
@montanaflynn Thank you for doing some testing! What browser are you using?

**Comment by montanaflynn** [comment]
@ryanto 

- Latest chrome
- Sometimes it happens the first time, the second time or any time. I thought maybe it was due to choosing the same file, but I can choose the same file 5 times in a row and it might work on the 3rd and 5th times, or first and last, it's pretty random. 
- I made a codesandbox and it works every time, but the exact same code locally is sporadic, and actually a few days ago was working every time.

https://codesandbox.io/s/sharp-shamir-d787sl?file=/src/App.js
